corehq/apps/ota/decorators.py

```diff
@@ -47,6 +47,11 @@ def _inner(request, domain, *args, **kwargs):
     return _inner
 
 
+def is_from_formplayer(request):
```

Comment:
I was hoping to integrate this into `require_mobile_access` (defined just above), but I couldn't come up with a clean way to do it and still return `HttpResponseForbidden` there